### PR TITLE
Adding containment and triage slas

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-3731)

## Description
Adding more SLA timers ( Triage and Containment ).
Edited the SLA dashboard accordingly.

## Screenshots
image

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bakatzir — COMMENTED** [review]
I suggest not having MTTD by type, but the full name of the widget(no synonym)
The by addition severity is also good.

Regarding the SLA value itself, we all know that it should be by severity, and this is an item I'm taking to later platform versions.

**Comment by nkanon** [comment]
I suggest making the 4 left widgets (Count by status) to present a shorter period of time (perhaps 7 days) as they are more operational. 

Sasha this looks great!

File: Packs/CommonDashboards/ReleaseNotes/1_2_2.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Dashboards
+##### SLA
+Added new widgets to the 'SLA' dashboard.
```
Comment (altmannyarden) [inline Packs/CommonDashboards/ReleaseNotes/1_2_2.md:4]: I would say something like The dashboard now support new triage and containment SLA's

Comment (ssokolovich) [reply Packs/CommonDashboards/ReleaseNotes/1_2_2.md]: Done

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**